### PR TITLE
Add SRT Ingest URL to Dashboard

### LINKED ISSUE #752: Add SRT Ingest URL to Dashboard
We now have SRT ingest! We should include the SRT ingest URL on the dashboard. 

SRT URLs look like this:

`srt://rtmp.livepeer.com:2935?streamid={stream-key}`

The stream key field should be auto-populated with the stream key. The stream key should be hidden on the dashboard view, as we do with the playback URL. However, when the copy button is clicked the full URL is used. 

Like this:

Tooltip should read:

> SRT ingest is currently in beta. Ensure that your encoder and player is compatible with SRT.

### PR DESCRIPTION
**Pull request by FullStackTiger**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

- Add SRT Ingest URL to Dashboard

**Does this pull request close any open issues?**

- Fixes #752 

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by adamsoffer — COMMENTED** [review]
I'm getting an "unexpected error" when navigating to the stream detail view.

**Comment by FullStackTiger** [comment]
> I'm getting an "unexpected error" when navigating to the stream detail view.

fixed @adamsoffer

File: packages/www/layouts/streamDetail.tsx [hunk 1]
```diff
@@ -458,6 +461,17 @@ const StreamDetail = ({
                       <Cell css={{ cursor: "pointer" }}>
                         <ShowURL url={globalIngestUrl} anchor={false} />
                       </Cell>
+                      <Cell css={{ color: "$mauve11" }}>SRT ingest URL</Cell>
+                      <Cell css={{ cursor: "pointer" }}>
+                        <ShowURL
+                          url={globalSrtIngestUrl}
+                          shortendUrl={globalSrtIngestUrl.replace(
+                            globalSrtIngestUrl.slice(38),
```
Comment (victorges) [inline packages/www/layouts/streamDetail.tsx:469]: Is this the `streamKey` length? WDYT of?:
```suggestion
                            globalSrtIngestUrl.slice(stream?.streamKey ?? 0),
```

File: packages/www/layouts/streamDetail.tsx [hunk 2]
```diff
@@ -267,6 +267,9 @@ const StreamDetail = ({
   const domain = isStaging() ? "monster" : "com";
   const globalIngestUrl = `rtmp://rtmp.livepeer.${domain}/live`;
   const globalPlaybackUrl = `https://cdn.livepeer.${domain}/hls/${playbackId}/index.m3u8`;
+  const globalSrtIngestUrl = `srt://rtmp.livepeer.${domain}:2935?streamid=${
+    stream?.streamKey || ""
```
Comment (victorges) [inline packages/www/layouts/streamDetail.tsx:271]: **nit:**
```suggestion
    stream?.streamKey ?? ""
```

**Review by victorges — APPROVED** [review]

**Pull request merged**